### PR TITLE
Introducing new tutorial on YOLOL

### PR DESCRIPTION
**Pull request by pascalstockert**
A first readme.md containing the structure of the series of tutorials

### CODE REVIEW

**Comment by thomasio101** [comment]
I haven't checked the spelling yet, but at a first glance this seems like a great start, @pascalstockert!

By the way, would you consider it to be a good idea if someone took this and implemented it as an interactive tutorial?

File: information/yolol_tutorial/readme.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# YOLOL Tutorial
+
+Welcome to this tutorial covering everything from the basics to optimization in YOLOL!<br>
+Over the course of the next few days and weeks new tutorials will be published not only here but on YouTube aswell!
+If you should have any questions arising while reading through the tutorial or watching the videos please feel free to 
+drop by in the [CYLON Discord Server](https://discord.gg/jkXcT74). We have a knowledgable group of people there that are
+always happy to answer!
```
Comment (martindevans) [inline information/yolol_tutorial/readme.md]: ... answer **questions**!

File: information/yolol_tutorial/readme.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+# YOLOL Tutorial
+
+Welcome to this tutorial covering everything from the basics to optimization in YOLOL!<br>
+Over the course of the next few days and weeks new tutorials will be published not only here but on YouTube aswell!
+If you should have any questions arising while reading through the tutorial or watching the videos please feel free to 
+drop by in the [CYLON Discord Server](https://discord.gg/jkXcT74). We have a knowledgable group of people there that are
```
Comment (martindevans) [inline information/yolol_tutorial/readme.md]: ...people there **who** are...

File: information/yolol_tutorial/readme.md (hunk 1 above)
Comment (pascalstockert) [reply information/yolol_tutorial/readme.md]: Corrected in new commit c;

File: information/yolol_tutorial/readme.md (hunk 2 above)
Comment (pascalstockert) [reply information/yolol_tutorial/readme.md]: Also corrected in new commit c;

**Comment by martindevans** [comment]
@pascalstockert has continued this work in a [separate repo](https://github.com/pascalstockert/yolol), so this PR is no longer necessary.

**Pull request closed without merging**